### DISCUSSION TITLE
Ok thought my RS485 issue was a Mega-Bas issue, But No?

### DISCUSSION THREAD

**Discussion by DrJeff** [tagyoureit/nodejs-poolController#712] [RS485] [answered]
I thought my issue with the RS485 was a Mega-BAS issue but I have no RS485 from Pumps tried changning addresses, adapters, disconnecting and reconecting, power cycling. I think it is a core issue.
[replay (2).zip](https://github.com/tagyoureit/nodejs-poolController/files/11175287/replay.2.zip)

I even reinstalled on a new Pi4, same RS485 problem with MegaBAS Board ver3.3 or 4.1 or USB adapter

**Comment by DrJeff** [comment, marked as answer]
Oh my! Not anything with code or software one pump had a bad bus wire was getting grounded. Changed bad cable all working now!  🥳

**Reply by rstrouse** [reply]
That will do it!
